### PR TITLE
DEV 2007 drop py35

### PR DESCRIPTION
**Pull request by qiaouchicago**
pyupgrade

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ language: python
 env:
   - ES_VERSION="7.7.1"
 python:
-  - "3.5"
```
Comment (wrkhenddher) [inline .travis.yml:6]: 😱😱😱😱

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ language: python
...
+  - "3.6"
   - "3.7"
```
Comment (wrkhenddher) [inline .travis.yml:7]: Could we go up higher? (3.8 or even 3.9)

**Review by wrkhenddher — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (qiaouchicago) [reply .travis.yml:7]: I checked and am waiting for this:
https://github.com/NCI-GDC/esbuild/blob/develop/.travis.yml

**Review by mpsolano — APPROVED** [review]
Looks good!

**Pull request merged**